### ISSUE TITLE
Report for data

### ISSUE THREAD

**Issue by sharlagelfand** [purposeanalytics/lemur#15]
Related to #13 but a single page report showing the charts from the sidebar (instead of just the data itself).

We should allow options for a PDF (because people like PDFs) and an HTML file (because PDFs are not accessible)
